flocx_market/api/bid.py

```diff
@@ -1,47 +1,36 @@
 from flask_restful import Resource
 from flask import request
-from flocx_market.db.sqlalchemy import api as dbapi
+import flocx_market.objects.bid as bid
 
 
 class Bid(Resource):
 
     @classmethod
     def get(cls, marketplace_bid_id=None):
-        if marketplace_bid_id is None:
-            return BidList.get()['bids']
```

Comment:
We still want this; only now it returns bid.Bid.get_all() (converted to dicts) explicitly.